boot/zephyr/Kconfig

```diff
@@ -13,6 +13,12 @@ config MCUBOOT
 	bool
 	select MPU_ALLOW_FLASH_WRITE if ARM_MPU
 
+# When compiling MCUBoot, the image will be linked to the boot partition.
+# Override .text offset to make sure it is set to zero.
+# This is necessary when other bootloaders set a different default.
```

Comment:
what a other bootloaders?

Reply:
You could conditionally do this based on `BOARD_HAS_NRF5_BOOTLOADER` perhaps? That way we don't pollute the rest

Reply:
I have address both comments.